jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -15,3 +38,5 @@ install_ringsh
 build_ring
 show_ring_status
 install_sproxyd
+set_keepalive $KEEPALIVE
```

Comment:
If you rely on this variable, then you should assert early that this variable is set and has a value.